stage1/stop/stop_fly.mk

```diff
@@ -0,0 +1 @@
+include stage1_fly/makelib/aci_binary.mk
```

Comment:
I think we should consider it with greater directories reorganization (I agree with https://github.com/coreos/rkt/pull/2438#issuecomment-211848536)